src/sass/components/_gallery.scss

```diff
@@ -1,68 +1,88 @@
 .gallery__title {
-    display: none;
+  display: none;
 }
 
 .gallery {
-    width: 100%;
-    min-height: 122px;
-    background-image: url("../images/gallery/home-pic5_320_122.png"),
-        linear-gradient(to bottom, #fff5f6 40%, #ffffff 40%);
-    background-size: cover;
-    background-position: center;
-    background-repeat: no-repeat;
+  width: 100%;
+  height: 122px;
+  background-image: url('../images/gallery/home-pic5_320_122.png'),
+    linear-gradient(to bottom, #fff5f6 40%, #ffffff 40%);
+  background-size: contain;
+  background-position: center;
+  background-repeat: no-repeat;
+  margin: 0 auto;
 }
 @media (min-device-pixel-ratio: 2), (min-resolution: 192dpi), (min-resolution: 2dppx) {
-    .gallery {
-        background-image: url("../images/gallery/home-pic5_640_244.png"),
-            linear-gradient(to bottom, #fff5f6 40%, #ffffff 40%);
-    }
+  .gallery {
+    background-image: url('../images/gallery/home-pic5_640_244.png'),
+      linear-gradient(to bottom, #fff5f6 40%, #ffffff 40%);
+  }
 }
 @media (min-device-pixel-ratio: 3), (min-resolution: 268dpi), (min-resolution: 3dppx) {
+  .gallery {
+    background-image: url('../images/gallery/home-pic5_960_366.png'),
+      linear-gradient(to bottom, #fff5f6 40%, #ffffff 40%);
+  }
+}
+@media (min-width: 320px) {
```

Comment:
tutaj lepiej aby miało jeszcze 
@media (min-width: 320px) and (max-with: 767px)  

ponieważ  wtedy się wyłączy a tak to będzie nadpisywane